src/app/components/Search.js

```diff
@@ -100,12 +100,12 @@ class Search extends Component {
     // }
 
     render() {
-        const { isFetching, images, existMoreItems } = this.props.requestService;
+        const { isFetching, images, existMoreItems, noImages } = this.props.requestService;
         const loader = isFetching && <Loader />;
-        // const noImages = this.props.noImages === true &&
-        //     (<div className="text-center mt-3">
-        //         <p>No images to display :(</p>
-        //     </div>);
+        const noImagesToShow = noImages &&
```

Comment:
is it necessary to create a state for it?
can you check noImages status by `!images.length`?

Reply:
My initial state has images.length === 0, so this message will be appear when page is loaded